webhook.go

```diff
@@ -145,7 +188,7 @@ func (w *Webhook) clean() error {
 	body := &WebhookResponseList{}
 
 	if err := json.NewDecoder(response.Body).Decode(body); err != nil {
-		log.Printf("Error decoding webhook list: %s\n", err)
+		log.Printf("Error decoding webhook list: %+v\n", response)
```

Comment:
Does this give a better logging or was this meant to be temporary?

Reply:
Yeah, this was just for logging. Pushed up a fix 👍